src/docker-in-docker/install.sh

```diff
@@ -561,6 +563,20 @@ dockerd_start="AZURE_DNS_AUTO_DETECTION=${AZURE_DNS_AUTO_DETECTION} DOCKER_DEFAU
         DEFAULT_ADDRESS_POOL="--default-address-pool $DOCKER_DEFAULT_ADDRESS_POOL"
     fi
 
+    if [ -z "$DOCKER_DEFAULT_IP6_TABLES" ]
+    then
+        DEFAULT_IP6_TABLES=true
+    else
+        DEFAULT_IP6_TABLES=$DOCKER_DEFAULT_IP6_TABLES
+    fi
+
+    # Create the daemon.json file with the provided value
+    cat <<JSON_EOF > /etc/docker/daemon.json
+{
+    "ip6tables": $DEFAULT_IP6_TABLES
```

Comment:
Instead of this, have you tried using `--ip6tables=<value>` same like DEFAULT_ADDRESS_POOL ?

Reply:
Pls check for the above mentioned method as a solution [here](https://github.com/devcontainers/features/pull/1068)

Reply:
If we have https://github.com/devcontainers/features/pull/1068 open, why do we need this PR which attempts to do the same thing?